.gitignore

```diff
@@ -12,3 +12,9 @@
 
 # Test coverage results.
 .coverage
+
+# Mac DS_Store files
+.DS_Store
```

Comment:
Please, remove DS_Store from here, it's usually not encouraged to include tmp files specific to your machine